openbb_platform/providers/fmp/openbb_fmp/models/government_trades.py

```diff
@@ -0,0 +1,162 @@
+"""Government Trades Model."""
+
+import asyncio
+import math
+from typing import Any, Dict, List, Optional
+from warnings import warn
+
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.government_trades import (
+    GovernmentTradesData,
+    GovernmentTradesQueryParams,
+)
+from openbb_core.provider.utils.errors import EmptyDataError
+from openbb_core.provider.utils.helpers import amake_request
+from openbb_fmp.utils.helpers import create_url, response_callback
+from pydantic import Field
+
+
+class FMPGovernmentTradesQueryParams(GovernmentTradesQueryParams):
+    """Government Trades Query Parameters.
+
+    Source: https://financialmodelingprep.com/api/v4/senate-trading?symbol=AAPL
+    Source: https://financialmodelingprep.com/api/v4/senate-trading-rss-feed?page=0
+    Source: https://financialmodelingprep.com/api/v4/senate-disclosure?symbol=AAPL
+    Source: https://financialmodelingprep.com/api/v4/senate-disclosure-rss-feed?page=0
+    """
+
+
+class FMPGovernmentTradesData(GovernmentTradesData):
+    """Government Trades Data Model."""
+
+    __alias_dict__ = {
+        "symbol": "ticker",
+        "transaction_date": "transactionDate",
+        "representative": "office",
+    }
+    link: Optional[str] = Field(
+        default=None, description="Link to the transaction document."
+    )
+    transaction_date: Optional[str] = Field(
```

Comment:
This is already defined in the standard model, we don't need this here. Additionally, the types do not match.